data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/PluginRepository.java

```diff
@@ -69,6 +70,26 @@ private static void findPlugins() {
         }
     }
 
+    private static PluginType extractPluginType(final DataPrepperPlugin dataPrepperPluginAnnotation) {
```

Comment:
This is a transitional method, right? We intend to remove it once we've moved to the new plugin model. Can we just comment that this will be removed in a later release?